### ISSUE TITLE
[question] CI running environment

### ISSUE THREAD

**Issue by jichenjc** [openshift/odo#2700]
[kind/Feature]

## Which functionality do you think we should add?
currently the CI travis is running with openshift v3.11 image hosted 
https://hub.docker.com/r/openshift/origin/tags

do we plan to move to openshift4 in the near future ?
```
OPENSHIFT_CLIENT_BINARY_URL=${OPENSHIFT_CLIENT_BINARY_URL:-'https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz'}
root@jck8s4:~/go/src/github.com/openshift/odo# vi scripts/oc-cluster.sh
root@jck8s4:~/go/src/github.com/openshift/odo# OPENSHIFT_CLIENT_VERSION=`echo $OPENSHIFT_CLIENT_BINARY_URL | awk -F '//' '{print $2}' | cut -d '/' -f 6`
root@jck8s4:~/go/src/github.com/openshift/odo# echo $OPENSHIFT_CLIENT_VERSION
v3.11.0

```

**Comment by amitkrout** [comment]
> do we plan to move to openshift4 in the near future ?

@jichenjc We have already configure our CI for 4.* cluster, but these 4.* cluster are not running on travis instances. We are using openshift prow for spinning up 4.* cluster on aws instances and then running out test against that cluster. Here is the job configuration location - https://github.com/openshift/release/tree/master/ci-operator/config/openshift/odo

So each pr is tested on 3.11 (travis instances) and 4.1, 4.2, 4.3 (prow instances).

See image for reference.

<img width="757" alt="Screen Shot 2020-03-11 at 12 58 07 PM" src="https://user-images.githubusercontent.com/19851115/76392654-fdf31800-6397-11ea-81bd-ee704565b95c.png">

or you can directly go to any pr and check the status there.

**Comment by jichenjc** [comment]
thanks for the info ,this is very helpful to me :)

close this issue~

**Comment by zhengxiaomei123** [comment]
@amitkrout Hi, I am working on the CI enablement for odo on OCP/s390x.  I found in your CI pipeline, there is 'Setting up pipeline imagestream for the test' job. Could you please show us the imagestream needed ? Thanks.

**Comment by jichenjc** [comment]
reopen due to 3.11 question need some help on this

**Comment by amitkrout** [comment]
> @amitkrout Hi, I am working on the CI enablement for odo on OCP/s390x. I found in your CI pipeline, there is 'Setting up pipeline imagestream for the test' job. Are those imagestreams releated to odo e2e testcases or not? If so, Could you please show us the imagestream needed ? Thanks.

@zhengxiaomei123 Thanks for the question.
TBH i could not get it when you mentioned ```there is 'Setting up pipeline imagestream for the test' job```, so can you please add some more reference from where/how you find it. Then it would be easy for me to answer your query

**Comment by zhengxiaomei123** [comment]
> TBH i could not get it when you mentioned `there is 'Setting up pipeline imagestream for the test' job`, so can you please add some more reference from where/how you find it. Then it would be easy for me to answer your query

@amitkrout  I got the information here [https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_odo/2759/pull-ci-openshift-odo-master-v4.2-integration-e2e-benchmark/1364/build-log.txt](build-log) . Thanks. As you mentioned above, there are default imagestreams or s2i builder image for 3.* cluster? What's about the 4.* cluster? I found there some code to import image for test, the url for this image are all based on amd-64. 
`oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("rhscl", "nodejs-8-rhel7:latest"), "nodejs:latest", project)`
So I think If I want to enable CI on OCP/Z 4.* cluster, I need to make those imagestream or s2i builder images ready for tests run.
